### ISSUE TITLE
I think moonlight is too out of date.

### ISSUE THREAD

**Issue by dexthecool** [southoz/RetroOZ#75]
After looking into my issue some more I realized that moonlight is very out of date.

**Comment by InfernoDigital** [comment]
Yeah, hope this can get updated soon in a new firmware version for RetroOz, I installed it specifically to do Rocket League streaming but any time I try to do an online match, the video stream freezes from Sunshine. I'm left with the ability to control the game on my pc monitor, but can't see it on the handheld anymore. Would love to see either this fixed, or some port of the Steam Remote Play to RetroOz so we can just use that instead.
